### PR TITLE
Add "joining" page

### PR DESCRIPTION
**Pull request by elly**
Add a first draft of the new "joining" page. This one will replace the in-game text, the wiki page, and (eventually) the forum post.

### CODE REVIEW

**Comment by ayust** [comment]
👍

File: joining.html [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    <p>
+      After your trial fleet, your recruiter will work with you to schedule
+      an interview over Teamspeak. This interview generally lasts half an
+      hour at most. You'll probably be asked about at least these things:
+      <ul>
+        <li>Your experiences in Eve</li>
+        <li>Your corp history</li>
+        <li>Your trial fleet experience</li>
+      </ul>
+    </p>
```
Comment (Calavoow) [inline joining.html:111]: What about the written interview?

File: joining.html [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+      can lead to a "bittervet" atmosphere of negativity about the game. That
+      isn't fun for anyone.
+    </p>
+    <p>
+      If you're over the 15M SP maximum, don't worry - you can still join, but
+      there's an extra step. Hop into our public channel or join a couple of
+      our regular public fleets, and try to get to know a couple of existing
+      members of the corp. We're looking for culture fit here a lot more than
+      pilot skill - it's fine if you're bad at the game, as long as you have a
+      good attitude about it.
```
Comment (Calavoow) [inline joining.html:169]: We'll need to schedule fleets more regularly or somehow communicate this information to interested parties if they would need to join multiple fleets.

File: joining.html [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+      <a name="questions" href="#questions">#</a>
+      Questions You Might Have
+    </h2>
+    <h3>
+      <a name="why-sp-max" href="#why-sp-max">#</a>
+      Why is there an SP maximum?
+    </h3>
+    <p>
+      Short answer: because we want SOUND to be a group focused on teaching
+      and learning.
```
Comment (Calavoow) [inline joining.html:154]: This doesn't seem related to the long answer and I don't think it's the right summary.

File: joining.html [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+      might want to learn more about.
+    </p>
+
+    <h3>
+      <a name="why-trial-fleet" href="#why-trial-fleet">#</a>
+      Why do I need to go on a fleet before I even join?
+    </h3>
+    <p>
+      Short answer: because we want to make sure you aren't annoying on
+      voice comms.
```
Comment (Calavoow) [inline joining.html:199]: Would suggest changing to "we want to see what you're like on voice comms."

File: joining.html [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+    </p>
+    <p>
+      Long answer: because we want to give you the chance to see how we
+      operate a bit, and to get a chance to see how you fly. We are definitely
+      <b>not</b> looking for any kind of serious PvP skill here, regardless of
+      how many SP you have! What we are looking for is:
+      <ul>
+        <li>Attitude: no whining, patience if things are slow, etc.</li>
+        <li>Comms discipline: knowing when it's important to speak and when
+            it's important to be quiet</li>
```
Comment (Calavoow) [inline joining.html:209]: This is quite difficult, even for vets!

**Review by Calavoow — COMMENTED** [review]
Some comments

File: joining.html (hunk 1 above)
Comment (elly) [reply joining.html:111]: Changed to clarify that it can be written, verbal, or both simultaneously, since I know some recruiters prefer one type or the other.

File: joining.html (hunk 2 above)
Comment (elly) [reply joining.html:169]: Yep, it's true. I have an idea for how to do that but it is not fleshed out yet - I need to talk to the recruiters some about it.

File: joining.html (hunk 3 above)
Comment (elly) [reply joining.html:154]: Rewritten - what do you think now?

File: joining.html (hunk 4 above)
Comment (elly) [reply joining.html:199]: Done!

File: joining.html (hunk 5 above)
Comment (elly) [reply joining.html:209]: Sure is :)

File: joining.html [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+      <a name="questions" href="#questions">#</a>
+      Questions You Might Have
+    </h2>
+    <h3>
+      <a name="why-sp-max" href="#why-sp-max">#</a>
+      Why is there an SP maximum?
+    </h3>
+    <p>
+      Short answer: because we want SOUND to be a positive, friendly place for
+      low-SP and new players.
```
Comment (Calavoow) [inline joining.html:155]:
```suggestion
      Short answer: because we want SOUND to be a positive, friendly place.
```

Maybe a bit too broad?

**Review by Calavoow — CHANGES_REQUESTED** [review]

File: joining.html (hunk 3 above)
Comment (Calavoow) [reply joining.html]: <mark outdated> (I don't know how)

File: joining.html (hunk 6 above)
Comment (elly) [reply joining.html:155]: yeah, I think that is a bit too broad - it doesn't really reference SP at all, which makes it a weird short answer to a question about an SP max :)

Comment (Calavoow) [reply joining.html:155]: But somehow it reflects on both our policies for <15M sp and >15M sp. Otherwise we need an answer for "why do I need a vouch when I have >15M sp?"

Comment (elly) [reply joining.html:155]: Hm, that's a good point. Okay, accepted :)
